activerecord/lib/active_record/model_schema.rb

```diff
@@ -524,9 +524,9 @@ def reset_column_information
       end
 
       def load_schema # :nodoc:
-        return if schema_loaded?
+        return false if schema_loaded?
```

Comment:
Why is this returning false now?

Reply:
To know if the schema was loaded or not. I didn't need it in the end, so I could revert this part.

Reply:
Is it to differentiate from `nil`? I can't see where `nil` could happen. If it is to match the `true` with `false` I think we should remove it, `nil` is false.

Reply:
It was just to be more explicit. But yes I'll remove it.